### ISSUE TITLE
Progression Reporting on console app.

### ISSUE THREAD

**Issue by Theoistic** [Microsoft/VisualStudioUninstaller#38]
ForcedUninstaller should show progress reports, Visual Studio takes a long time to install, presumably the same goes for uninstalling. After running for 20+ minutes, it would be nice to know if its stuck in a while(true) {} or not.
Does not have to be deep progression, just to indicate that it is still working without having a CPU profiler running side by side.

**Comment by tobyhu87** [comment]
Thank you for the suggestion.  

We could have the Uninstaller connect to the Visual Studio setup's burn pipe.

**Comment by XanNava** [comment]
Yes please please add this, so that way people know that it is doing something actually...

**Comment by nektobit** [comment]
agree
